mdtraj/utils/delay_import.py

```diff
@@ -78,15 +78,15 @@
     sharing of array-oriented scientific data.
 
     netcdf4-python can be downloaded from https://pypi.python.org/pypi/netCDF,
```

Comment:
IIUC, just based on following links, this should be 
```suggestion
    netcdf4-python can be downloaded from https://pypi.python.org/pypi/netCDF4,
```